interfaces/devdoc/threadpool_requirements.md

```diff
@@ -139,18 +139,16 @@ MOCKABLE_FUNCTION(, int, threadpool_schedule_work_item, THANDLE(THREADPOOL), thr
 ### threadpool_destroy_work_item
 
 ```c
-MOCKABLE_FUNCTION(, void, threadpool_destroy_work_item, THANDLE(THREADPOOL), threadpool, THREADPOOL_WORK_ITEM_HANDLE, work_item_context);
+static void threadpool_destroy_work_item (THANDLE(THREADPOOL_WORK_ITEM) work_item_context);
 ```
 
-`threadpool_destroy_work_item` closes and frees all the member variables in the context of type `THREADPOOL_WORK_ITEM_HANDLE` and then frees the context
-
-**NON_THREADPOOL_05_011: [ ** If `threadpool` is `NULL`, `threadpool_destroy_work_item` shall fail and return a non-zero value. **]**
+`threadpool_destroy_work_item` frees the threadpool work item context.
 
-**NON_THREADPOOL_05_012: [** If `work_item_context` is `NULL`, `threadpool_destroy_work_item` shall fail and not do anything before returning. **]**
+**NON_THREADPOOL_05_011: [** If `work_item_context` is `NULL`, `threadpool_destroy_work_item` shall fail and not do anything before returning. **]**
 
-**NON_THREADPOOL_05_013: [** `threadpool_destroy_work_item` shall close and free all the members of `THREADPOOL_WORK_ITEM_HANDLE`. **]**
+**NON_THREADPOOL_05_012: [** `threadpool_destroy_work_item` shall perform cleanup and free all the members of `THREADPOOL_WORK_ITEM_HANDLE`. **]**
 
-**NON_THREADPOOL_05_014: [** `threadpool_destroy_work_item` shall free the `work_item_context` of type `THREADPOOL_WORK_ITEM_HANDLE`. **]**
+**NON_THREADPOOL_05_013: [** `threadpool_destroy_work_item` shall free the `work_item_context` of type `THREADPOOL_WORK_ITEM_HANDLE`. **]**
```

Comment:
Same comments like for linux and win32, please look at the other 2 files.

Reply:
Done